plugins/hypervisors/kvm/src/test/java/com/cloud/hypervisor/kvm/resource/LibvirtComputingResourceTest.java

```diff
@@ -4403,68 +4381,23 @@ public void testPvlanSetupCommandVm() {
         assertFalse(answer.getResult());
     }
 
-    @SuppressWarnings("unchecked")
-    @Test
-    public void testPvlanSetupCommandDhcpException() {
```

Comment:
Since the old way used to call the libvirt library, it could potentially throw a LibvirtException. Now that it doesn't, theres no need to catch or test whether the exception has occoured